source/code/plugins/oms_configuration.rb

```diff
@@ -365,6 +365,15 @@ def set_request_intervals(topology_interval, telemetry_interval)
         OMS::Log.info_once("OMS agent management service telemetry request interval now #{@@TelemetryInterval}")
       end
 
+      # Check whether DSC has been disabled (which prevents automatic config management)
+      def is_dsc_disabled
+        begin
+          return File.exist?("/etc/opt/omi/conf/omsconfig/omshelper_disable")
+        rescue => e
+          OMS::Log.error_once("Failed to get DSC enabled/disabled status. #{e}")
```

Comment:
We do not return a boolean to caller on exception. How would this effect the flow ?